book/src/group-snapshot.md

```diff
@@ -0,0 +1,119 @@
+# Volume Group Snapshot Feature
+
+## Status
+
+Status | Min K8s Version | Max K8s Version | snapshot-controller Version | snapshot-validation-webhook Version | CSI external-snapshotter sidecar Version | external-provisioner Version
+--|--|--|--|--|--|--
+Alpha  | 1.27 | - | 7.0+ | 7.0+ | 7.0+ | 1.8+
+
+## Overview
+
+Some storage systems provide the ability to create a crash consistent snapshot of
+multiple volumes. A group snapshot represents “copies” from multiple volumes that
+are taken at the same point-in-time. A group snapshot can be used either to rehydrate
+new volumes (pre-populated with the snapshot data) or to restore existing volumes to
+a previous state (represented by the snapshots).
+
+Kubernetes CSI currently enables CSI Drivers to expose the following functionality via the Kubernetes API:
+
+1. Creation and deletion of volume group snapshots via [Kubernetes native API](https://github.com/kubernetes-csi/external-snapshotter/tree/master/client/apis/volumegroupsnapshot/v1alpha1). 
+2. Creation of new volumes pre-populated with the data from a snapshot that is part of the volume group snapshot via Kubernetes [dynamic volume provisioning](https://kubernetes.io/docs/concepts/storage/dynamic-provisioning/).
+
+## Implementing Volume Group Snapshot Functionality in Your CSI Driver
+
+To implement the volume group snapshot feature, a CSI driver MUST:
+
+* Implement a new group controller service.
+* Implement group controller RPCs: `CreateVolumeGroupSnapshot`, `DeleteVolumeGroupSnapshot`, and `GetVolumeGroupSnapshot`.
+* Add group controller capability `CREATE_DELETE_GET_VOLUME_GROUP_SNAPSHOT`.
+
+For details,  see the [CSI spec](https://github.com/container-storage-interface/spec/blob/master/spec.md).
+
+## Sidecar Deployment
+
+The Kubernetes CSI development team maintains the [external-snapshotter](external-snapshotter.md) Kubernetes CSI [Sidecar Containers](sidecar-containers.md). This sidecar container implements the logic for watching the Kubernetes API objects and issuing the appropriate CSI volume group snapshot calls against a CSI endpoint. For more details, see [external-snapshotter documentation](external-snapshotter.md).
+
+## Volume Group Snapshot APIs
+
+With the introduction of Volume Group Snapshot, the user can create and delete a group snapshot using Kubernetes APIs. 
+
+The schema definition for the custom resources (CRs) can be found [here](https://github.com/kubernetes-csi/external-snapshotter/tree/master/client/config/crd). The CRDs should be installed by the Kubernetes distributions.
+
+There are 3 new APIs:
```

Comment:
Watch out for “new”. Documentation likes to be [timeless](https://developers.google.com/style/timeless-documentation).

Reply:
Removed "new".